### ISSUE TITLE
Is the roslyn Sublime currently working?

### ISSUE THREAD

**Issue by dkdewitt** [OmniSharp/omnisharp-sublime#223]
I have tried building and running Roslyn with Sublime a few ways and am wondering if this is currently working on OSX?

**Comment by RayRay26** [comment]
Bumping this since I'd thought to have a try at reviving this branch. I figured since all the devs migrated over to VSCode and left Sublime Text completely in the dust, I'd try to migrate the code in the Roslyn branch for Sublime to use. However I cannot get it to work. I spent a few hours building Omnisharp-Roslyn which was updated 3 days before this post. Then I tried setting my config file's "MSBuild" tag to the .msbuild folder that resulted:
`C:\\Users\\Aaron\\Desktop\\omnisharp-roslyn-master\\.msbuild\\15.0`
But I have absolutely no feedback on whether or not it is using this folder. Then I tried replacing the PrebuiltOmnisharpSever, but I have no idea how to get it to work; the Roslyn version of omnisharp.exe crashes about a second after the project starts loading.

If this branch is going to stop being updated, then how would I update Omnisharp, at least personally? All I know is that this throws an error:
`public int RealValue { get; private set; } = 10;`
An Auto-Property with an initialized value.
`Error : Unexpected symbol '='' in class, struct, or interface member declaration`
